src/client/js/app.js

```diff
@@ -7,12 +7,18 @@ let baseURLWeatherbitCurrent = 'http://api.weatherbit.io/v2.0/current?';
 let baseURLWeatherbitDaily = 'http://api.weatherbit.io/v2.0/forecast/daily?';
 let apiKeyWeatherbit = '4f64fed98275458e9ed0a797ec81774e';
 
-let baseURLPixabay = 'https://pixabay.com/api/?image_type=photo&category=buildings&q=';
+let baseURLPixabay = 'https://pixabay.com/api/?image_type=photo&category=travel&q=';
 let apiKeyPixabay = '6315616-d5cb7351229c7679827eaf034';
 
 // Object with all data
 let primaryData;
 
+let minDate = formatISO(startOfTomorrow(), { representation: 'date' });
+document.querySelector('#departure-date').setAttribute('min', minDate);
+
+let maxDate = formatISO(addDays(parseISO(minDate), 14), { representation: 'date' });
+document.querySelector('#departure-date').setAttribute('max', maxDate);
```

Comment:
It'd be nice to extract `document.querySelector('#departure-date')` into a variable `departureDateElement` so you won't need to search for the same element twice.